### PR TITLE
Add extra check on SingleServerApplianceToImage function in case the osFamily is null

### PR DESCRIPTION
**Pull request by alibazlamit**
1&1 allows uploading custom images therefor some of the images might have a null value for the osFamily. The necessary check was added, please merge.

Thanks!

### CODE REVIEW

**Comment by andreaturli** [comment]
lgtm

File: oneandone/src/main/java/org/apache/jclouds/oneandone/rest/compute/function/SingleServerApplianceToImage.java [hunk 1]
```diff
@@ -74,7 +74,11 @@ static boolean is64Bit(int architecture) {
       return tryFind(asList(OsFamily.values()), new Predicate<OsFamily>() {
          @Override
          public boolean apply(OsFamily input) {
-            return osFamily.toLowerCase().contains(input.value().toLowerCase());
+            if (osFamily != null) {
```
Comment (nacx) [inline oneandone/src/main/java/org/apache/jclouds/oneandone/rest/compute/function/SingleServerApplianceToImage.java]: `osFamily` is the input parameter of the function. There's no need to do the same redundant check in each iteration. Remove this and directly return absent at the beginning if the input parameter is null. Same for the other function.

**Comment by alibazlamit** [comment]
Changes applied.
Thanks!

**Comment by nacx** [comment]
Can you please squash the commits?

**Comment by alibazlamit** [comment]
Squashed .

**Comment by nacx** [comment]
Pushed to [master](http://git-wip-us.apache.org/repos/asf/jclouds-labs/commit/3b8f5c60) and [2.0.x](http://git-wip-us.apache.org/repos/asf/jclouds-labs/commit/f294ab10). Thanks @alibazlamit!

**Pull request closed without merging**